app/src/main/java/com/davidtomas/taskyapp/features/agenda/presentation/eventDetail/components/AgendaTypeComposable.kt

```diff
@@ -0,0 +1,71 @@
+package com.davidtomas.taskyapp.features.agenda.presentation.eventDetail.components
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.CornerRadius
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.davidtomas.taskyapp.coreUi.TaskyAppTheme
+
+@Composable
+fun AgendaTypeComposable(
+    title: String,
+    rectangleColor: Color,
+) {
+    Row(
+        modifier = Modifier.fillMaxWidth(),
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Canvas(modifier = Modifier.size(20.dp)) {
+            drawRoundRect(
+                color = rectangleColor,
+                size = androidx.compose.ui.geometry.Size(
+                    width = 20.dp.toPx(),
+                    height = 20.dp.toPx()
+                ),
+                cornerRadius = CornerRadius(
+                    x = 2.dp.toPx(),
+                    y = 2.dp.toPx()
+                )
+            )
+        }
```

Comment:
Couldn't this just be done with a simple modifier?

Reply:
I agree, Its just I wanted to use Canvas. I change it hahah